.github/workflows/publish-menifest.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish flyte manifest 
+on:
```

Comment:
Can this be triggered manually too? or maybe we add an additional manual workflow that just creates the tag?